mitxgraders/listgrader.py

```diff
@@ -692,6 +696,20 @@ def check_response(self, answer, student_input, **kwargs):
         answers = answer  # Rename from the ItemGrader name
         student_list = student_input.split(self.config['delimiter'])
 
+        # Check for empty entries in the list
+        if self.config['missing_error']:
+            bad_items = []
```

Comment:
`bad_items = [idx+1 for (idx, item) in enumerate(student_list) if item.strip() == '']` if you want.

Reply:
Yeah, that is neater.